Cargo.toml

```diff
@@ -5,3 +5,12 @@ description = "Correct by construction non-empty vector"
 authors = ["Alexis Sellier <self@cloudhead.io>"]
 edition = "2018"
 license = "MIT"
+
+[dependencies]
+serde = { features = ["serde_derive"], optional = true, version = "1.0.111" }
```

Comment:
I think a looser version bound would be better, since this is a library, eg. `version = "1"`

Reply:
agreed